webpack.config.babel.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+const webpack = require('webpack');
+const nodeExternals = require('webpack-node-externals');
+const BabiliPlugin = require('babili-webpack-plugin');
+const DotenvPlugin = require('dotenv-webpack');
+const path = require('path');
+
+const config: webpack.Configuration = {
```

Comment:
Wow this `webpack.Configuration` type is new to me. `webpack` provides this flowtype definition?